packages/static-module-record/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@endo/static-module-record",
-  "version": "0.4.1+1-dev",
+  "version": "0.5.0",
```

Comment:
✅ This is a bit of an edge case, but accidentally correct. The package has been renamed so we could technically restart the version count, but we’re preserving the monotonic ascent of version numbers in the curated `NEWS.md`.

Reply:
You added features.